.github/workflows/master.yml

```diff
@@ -256,15 +256,15 @@ jobs:
         comment: [ "normal" ]
         include:
           - java: 8
-            flink: '1.17'
+            flink: '1.20'
             flink-archive: '-Dflink.archive.mirror=https://archive.apache.org/dist/flink/flink-1.18.1 -Dflink.archive.name=flink-1.18.1-bin-scala_2.12.tgz'
             comment: 'verify-on-flink-1.18-binary'
           - java: 8
-            flink: '1.17'
+            flink: '1.20'
             flink-archive: '-Dflink.archive.mirror=https://archive.apache.org/dist/flink/flink-1.19.1 -Dflink.archive.name=flink-1.19.1-bin-scala_2.12.tgz'
             comment: 'verify-on-flink-1.19-binary'
           - java: 8
-            flink: '1.17'
+            flink: '1.20'
             flink-archive: '-Dflink.archive.mirror=https://archive.apache.org/dist/flink/flink-1.20.0 -Dflink.archive.name=flink-1.20.0-bin-scala_2.12.tgz'
             comment: 'verify-on-flink-1.20-binary'
```

Comment:
now we build the engine with flink 1.20, the binary check list should be 1.17, 1.18 and 1.19

Reply:
fixed